frontends/web/src/routes/buy/info.tsx

```diff
@@ -75,19 +75,6 @@ class BuyInfo extends Component<Props, State> {
       .catch(console.error);
   };
 
-  private getCryptoName = (): string => {
-    const { accounts, code, t } = this.props;
-    if (!code) {
-      const onlyBitcoin = accounts.every(({ coinCode }) => isBitcoinOnly(coinCode));
-      return onlyBitcoin ? 'Bitcoin' : t('buy.info.crypto');
-    }
-    const account = accounts.find(account => account.code === code);
-    if (account) {
-      return isBitcoinOnly(account.coinCode) ? 'Bitcoin' : t('buy.info.crypto');
-    }
-    return t('buy.info.crypto');
-  };
-
   public render() {
     const { t } = this.props;
```

Comment:
nit: destruct `accounts` and `code` here, so you can use those variables instead of `this.props.accounts` and `this.props.code`

Reply:
No need for destructing variables, I think. Just used `const { t, accounts, code } = this.props;`

Reply:
nice 👍